slowly-changing-lookup-cache/bin/send_message.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I think this file is redundant for the same reason above.

Reply:
it is common for shell scripts. :)